contracts/misc/marketplaces/SeaportAdapter.sol

```diff
@@ -24,42 +24,41 @@ contract SeaportAdapter is IMarketplace {
 
     function getAskOrderInfo(bytes memory params)
         external
-        view
+        pure
         override
         returns (DataTypes.OrderInfo memory orderInfo)
     {
-        (
-            AdvancedOrder memory advancedOrder,
-            CriteriaResolver[] memory resolvers,
-            ,
-            address recipient
-        ) = abi.decode(
-                params,
-                (AdvancedOrder, CriteriaResolver[], bytes32, address)
-            );
+        (AdvancedOrder[] memory advancedOrders, , ) = abi.decode(
+            params,
+            (AdvancedOrder[], CriteriaResolver[], Fulfillment[])
```

Comment:
For saving gas, if we don't need CriteriaResolver and Fulfillment, we don't need to decode them.